test/create.test.js

```diff
@@ -33,9 +33,7 @@ describe('create', function() {
       price: {type: Number},
     }, {forceId: false});
 
-    db.once('connected', function() {
-      db.automigrate(done);
-    });
+    db.automigrate(done);
```

Comment:
@ssh24 I believe some places don't run automigrate, could you give me an example?